camera_pipewire compiler errors/warnings

```
In file included from /mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/camera_pipewire/camera_plugin.cc:17:
In file included from /mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/camera_pipewire/camera_plugin.h:26:
/mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/camera_pipewire/CameraStream.h:131:8: warning: extra tokens at end of #endif directive [-Wextra-tokens]
  131 | #endif PLUGINS_CAMERA_PIPEWIRE_CAMERASTREAM_H_
      |        ^
      |        //
/mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/camera_pipewire/camera_plugin.cc:21:10: fatal error: 'glib.h' file not found
   21 | #include <glib.h>
      |          ^~~~~~~~
1 warning and 1 error generated.
[163/213] Building CXX object plugins/camera_pipewire/CMakeFiles/plugin_camera_pipewire.dir/camera_plugin_c_api.cc.o
In file included from /mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/camera_pipewire/camera_plugin_c_api.cc:18:
In file included from /mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/camera_pipewire/camera_plugin.h:26:
/mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/camera_pipewire/CameraStream.h:131:8: warning: extra tokens at end of #endif directive [-Wextra-tokens]
  131 | #endif PLUGINS_CAMERA_PIPEWIRE_CAMERASTREAM_H_
      |        ^
      |        //
1 warning generated.
[169/213] Building CXX object plugins/camera_pipewire/CMakeFiles/plugin_camera_pipewire.dir/CameraStream.cc.o
In file included from /mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/camera_pipewire/CameraStream.cc:17:
/mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/camera_pipewire/CameraStream.h:131:8: warning: extra tokens at end of #endif directive [-Wextra-tokens]
  131 | #endif PLUGINS_CAMERA_PIPEWIRE_CAMERASTREAM_H_
      |        ^
      |        //
1 warning generated.
[179/213] Building CXX object plugins/CMakeFiles/plugins.dir/mnt/raid10/work...escreen/shell/platform/homescreen/platform_views/platform_views_handler.cc.o
ninja: build stopped: subcommand failed.
Traceback (most recent call last):
```

--- a/plugins/camera_pipewire/CMakeLists.txt
+++ b/plugins/camera_pipewire/CMakeLists.txt
@@ -1,5 +1,5 @@
 #
-# Copyright 2023-2024 Toyota Connected North America
+# Copyright 2023-2025 Toyota Connected North America
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
@@ -22,10 +22,10 @@ add_library(plugin_camera_pipewire STATIC
         camera_plugin_c_api.cc
         camera_plugin.cc
         messages.g.cc
-        CameraStream.cc
-        CameraStream.h
-        PipewireGraph.cc
-        PipewireGraph.h
+        camera_stream.cc
+        camera_stream.h
+        pipewire_graph.cc
+        pipewire_graph.h
 )
 
 target_include_directories(plugin_camera_pipewire PUBLIC

--- a/plugins/camera_pipewire/camera_plugin.cc
+++ b/plugins/camera_pipewire/camera_plugin.cc
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020-2024 Toyota Connected North America
+ * Copyright 2020-2025 Toyota Connected North America
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -15,16 +15,21 @@
  */
 
 #include "camera_plugin.h"
+
+#include <memory>
+#include <string>
+#include <vector>
+
+extern "C" {
+#include <glib-2.0/glib.h>
+};
+#include <jpeglib.h>
+
 #include <flutter/event_stream_handler_functions.h>
 #include <flutter/plugin_registrar_homescreen.h>
 #include <flutter/standard_method_codec.h>
-#include <glib.h>
-#include <jpeglib.h>
-#include <memory>
-#include <string>
-#include <unordered_map>
-#include <vector>
-#include "PipewireGraph.h"
+
+#include "pipewire_graph.h"
 #include "plugins/common/common.h"
 
 extern "C" {
@@ -73,7 +78,7 @@ void CameraPlugin::RegisterWithRegistrar(
 CameraPlugin::CameraPlugin(flutter::PluginRegistrarDesktop* plugin_registrar,
                            flutter::BinaryMessenger* /*messenger*/)
     : mPreview(), registrar_(plugin_registrar) {
-  if (!PipewireGraph::instance().initialize()) {
+  if (!pipewire_graph::instance().initialize()) {
     spdlog::error("failed to initialize PipeWire manager!");
   }
 
@@ -110,12 +115,12 @@ CameraPlugin::CameraPlugin(flutter::PluginRegistrarDesktop* plugin_registrar,
 }
 
 CameraPlugin::~CameraPlugin() {
-  PipewireGraph::instance().shutdown();
+  pipewire_graph::instance().shutdown();
 }
 
 ErrorOr<flutter::EncodableList> CameraPlugin::GetAvailableCameras() {
   flutter::EncodableList list;
-  const PipewireGraph& mgr = PipewireGraph::instance();
+  const pipewire_graph& mgr = pipewire_graph::instance();
 
   const auto cameras = mgr.getCameraNodes();
   for (auto camera : cameras) {
@@ -134,11 +139,12 @@ void CameraPlugin::Create(
   spdlog::debug("[camera_plugin] create camera_id: {}", camera_id);
   if (CameraId_CameraStream.find(camera_id) == CameraId_CameraStream.end()) {
     auto new_camera =
-        std::make_shared<CameraStream>(registrar_, camera_id, 640, 480);
+        std::make_shared<camera_stream>(registrar_, camera_id, 640, 480);
 
     new_camera->on_image_frame =
-        [this](const uint8_t* y, int ys, const uint8_t* u_or_uv, int us,
-               const uint8_t* v, int vs, int w, int h, const char* raw) {
+        [this](const uint8_t* y, const int ys, const uint8_t* u_or_uv,
+               const int us, const uint8_t* v, const int vs, const int w,
+               const int h, const char* raw) {
           if (!image_stream_active_ || !image_sink_)
             return;
 

--- a/plugins/camera_pipewire/camera_plugin.h
+++ b/plugins/camera_pipewire/camera_plugin.h
@@ -23,7 +23,7 @@
 #include <flutter/plugin_registrar.h>
 #include <flutter/plugin_registrar_homescreen.h>
 
-#include "CameraStream.h"
+#include "camera_stream.h"
 #include "event_channel.h"
 #include "messages.g.h"
 #include "plugins/common/common.h"
@@ -119,8 +119,8 @@ class CameraPlugin final : public flutter::Plugin, public CameraApi {
   std::map<std::string, std::unique_ptr<flutter::StreamHandler<>>>
       stream_handlers_;
 
-  std::map<std::string, std::shared_ptr<CameraStream>> CameraId_CameraStream;
-  std::map<GLuint, std::shared_ptr<CameraStream>> TextureId_CameraStream;
+  std::map<std::string, std::shared_ptr<camera_stream>> CameraId_CameraStream;
+  std::map<GLuint, std::shared_ptr<camera_stream>> TextureId_CameraStream;
 
   std::unique_ptr<flutter::EventChannel<flutter::EncodableValue>>
       image_channel_;

--- a/plugins/camera_pipewire/camera_stream.cc
+++ b/plugins/camera_pipewire/camera_stream.cc
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020-2024 Toyota Connected North America
+ * Copyright 2020-2025 Toyota Connected North America
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-#include "CameraStream.h"
+#include "camera_stream.h"
 
 #include <cstdio>
 #include <filesystem>
@@ -30,7 +30,7 @@
 #include <spa/param/video/raw.h>
 #include <spa/pod/builder.h>
 
-#include "PipewireGraph.h"
+#include "pipewire_graph.h"
 #include "plugins/common/common.h"
 
 static constexpr char kPictureCaptureExtension[] = "jpeg";
@@ -130,10 +130,10 @@ static int decode_yuy2(const uint8_t* input,
 //------------------------------------------------------------------------------
 // Constructor
 //------------------------------------------------------------------------------
-CameraStream::CameraStream(flutter::PluginRegistrarDesktop* plugin_registrar,
-                           std::string camera_id,
-                           const int width,
-                           const int height)
+camera_stream::camera_stream(flutter::PluginRegistrarDesktop* plugin_registrar,
+                             std::string camera_id,
+                             const int width,
+                             const int height)
     : registrar_(plugin_registrar),
       width_(width),
       height_(height),
@@ -214,16 +214,16 @@ CameraStream::CameraStream(flutter::PluginRegistrarDesktop* plugin_registrar,
 //------------------------------------------------------------------------------
 // Destructor
 //------------------------------------------------------------------------------
-CameraStream::~CameraStream() {
+camera_stream::~camera_stream() {
   Stop();
 }
 
 //------------------------------------------------------------------------------
 // Start capturing from the given node ID
 //------------------------------------------------------------------------------
-bool CameraStream::Start(const std::string& camera_id) {
+bool camera_stream::Start(const std::string& camera_id) {
   // 1) Ensure the manager is running
-  auto& mgr = PipewireGraph::instance();
+  auto& mgr = pipewire_graph::instance();
   if (!mgr.initialize()) {
     spdlog::error("[CameraStream] fail to initialize PipewireGraph.");
     return false;
@@ -339,12 +339,12 @@ bool CameraStream::Start(const std::string& camera_id) {
 //------------------------------------------------------------------------------
 // Stop capturing
 //------------------------------------------------------------------------------
-void CameraStream::Stop() {
+void camera_stream::Stop() {
   if (!pw_stream_) {
     return;  // already stopped
   }
 
-  auto& mgr = PipewireGraph::instance();
+  auto& mgr = pipewire_graph::instance();
   auto* loop = mgr.threadLoop();
 
   // Lock while destroying
@@ -459,7 +459,7 @@ static void YUY2ToI420Planes(const uint8_t* src,
 //------------------------------------------------------------------------------
 // Private method: called each time there's a new MJPEG frame
 //------------------------------------------------------------------------------
-void CameraStream::HandleProcess() {
+void camera_stream::HandleProcess() {
   if (!pw_stream_)
     return;
   pw_buffer* buf = pw_stream_dequeue_buffer(pw_stream_);
@@ -570,25 +570,25 @@ const char* StreamStateToString(const pw_stream_state state) {
   }
 }
 
-void CameraStream::OnStreamStateChanged(void* /*data*/,
-                                        const pw_stream_state old_state,
-                                        const pw_stream_state new_state,
-                                        const char* /*error*/) {
+void camera_stream::OnStreamStateChanged(void* /*data*/,
+                                         const pw_stream_state old_state,
+                                         const pw_stream_state new_state,
+                                         const char* /*error*/) {
   spdlog::debug("[CameraStream] stream state changed from {} to {}",
                 StreamStateToString(old_state), StreamStateToString(new_state));
 }
 
-void CameraStream::OnStreamProcess(void* data) {
-  auto* self = static_cast<CameraStream*>(data);
+void camera_stream::OnStreamProcess(void* data) {
+  auto* self = static_cast<camera_stream*>(data);
   (void)self;
   self->HandleProcess();
 }
 
-void CameraStream::PauseStream() const {
+void camera_stream::PauseStream() const {
   if (!pw_stream_)
     return;
 
-  auto& mgr = PipewireGraph::instance();
+  auto& mgr = pipewire_graph::instance();
   if (!mgr.initialize()) {
     spdlog::error("[CameraStream] failed to initialize PipewireGraph.");
     return;
@@ -605,11 +605,11 @@ void CameraStream::PauseStream() const {
   pw_thread_loop_unlock(loop);
 }
 
-void CameraStream::ResumeStream() const {
+void camera_stream::ResumeStream() const {
   if (!pw_stream_)
     return;
 
-  auto& mgr = PipewireGraph::instance();
+  auto& mgr = pipewire_graph::instance();
   if (!mgr.initialize()) {
     spdlog::error("[CameraStream] failed to initialize PipewireGraph.");
     return;
@@ -625,7 +625,7 @@ void CameraStream::ResumeStream() const {
   { pw_stream_set_active(pw_stream_, true); }
   pw_thread_loop_unlock(loop);
 }
-std::optional<std::string> CameraStream::GetFilePathForPicture() {
+std::optional<std::string> camera_stream::GetFilePathForPicture() {
   std::ostringstream oss;
   oss << "xdg-user-dir PICTURES";
   std::string picture_path;
@@ -640,7 +640,7 @@ std::optional<std::string> CameraStream::GetFilePathForPicture() {
   return path;
 }
 
-std::string CameraStream::takePicture() const {
+std::string camera_stream::takePicture() const {
   auto filename = GetFilePathForPicture();
   save_image_to_jpeg(filename.value(), decoded_buffer_.get(), width_, height_,
                      3, 90);

--- a/plugins/camera_pipewire/camera_stream.h
+++ b/plugins/camera_pipewire/camera_stream.h
@@ -33,7 +33,7 @@
  * CameraStream manages a single PipeWire MJPEG camera stream and its Flutter
  * texture.
  */
-class CameraStream {
+class camera_stream {
  public:
   /**
    * Create a new CameraStream.
@@ -43,15 +43,15 @@ class CameraStream {
    * @param width      Desired width of the MJPEG frames.
    * @param height     Desired height of the MJPEG frames.
    */
-  CameraStream(flutter::PluginRegistrarDesktop* plugin_registrar,
-               std::string camera_id,
-               int width,
-               int height);
+  camera_stream(flutter::PluginRegistrarDesktop* plugin_registrar,
+                std::string camera_id,
+                int width,
+                int height);
 
   /**
    * Destructor. Automatically stops the camera stream if running.
    */
-  ~CameraStream();
+  ~camera_stream();
 
   /**
    * Start capturing from the given PipeWire node ID (camera).
@@ -128,4 +128,4 @@ class CameraStream {
   static void OnStreamProcess(void* data);
 };
 
-#endif PLUGINS_CAMERA_PIPEWIRE_CAMERASTREAM_H_
+#endif  // PLUGINS_CAMERA_PIPEWIRE_CAMERASTREAM_H

--- a/plugins/camera_pipewire/pipewire_graph.cc
+++ b/plugins/camera_pipewire/pipewire_graph.cc
@@ -1,40 +1,53 @@
+/*
+ * Copyright 2020-2025 Toyota Connected North America
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
-#include "PipewireGraph.h"
+#include "pipewire_graph.h"
 
 #include <spdlog/spdlog.h>
 
-#include <iostream>
-
-PipewireGraph& PipewireGraph::instance() {
-  static PipewireGraph s_instance;
+pipewire_graph& pipewire_graph::instance() {
+  static pipewire_graph s_instance;
   return s_instance;
 }
 
-PipewireGraph::PipewireGraph() = default;
+pipewire_graph::pipewire_graph() = default;
 
-PipewireGraph::~PipewireGraph() {
-  // Ensure shutdown is called in case user forgot
+pipewire_graph::~pipewire_graph() {
+  // Ensure shutdown is called in case the user forgot
   if (initialized_) {
     shutdown();
   }
 }
 
 // Callback function for detecting cameras
-void PipewireGraph::on_global(void* data,
-                              const uint32_t id,
-                              uint32_t /*permissions*/,
-                              const char* type,
-                              uint32_t version,
-                              const struct spa_dict* props) {
+void pipewire_graph::on_global(void* data,
+                               const uint32_t id,
+                               uint32_t /*permissions*/,
+                               const char* type,
+                               const uint32_t version,
+                               const struct spa_dict* props) {
   if (!data) {
-    std::cerr << "[Error] on_global received null data\n";
+    spdlog::error("on_global received null data");
     return;
   }
 
   if (!props)
     return;
 
-  auto* self = static_cast<PipewireGraph*>(data);
+  auto* self = static_cast<pipewire_graph*>(data);
 
   if (strcmp(type, PW_TYPE_INTERFACE_Node) == 0) {
     self->handleNodeInfo(id, version, props);
@@ -44,12 +57,12 @@ void PipewireGraph::on_global(void* data,
     self->handleLinkInfo(id, version, props);
   }
 }
-void PipewireGraph::on_global_remove(void* data, const uint32_t id) {
+void pipewire_graph::on_global_remove(void* data, const uint32_t id) {
   if (!data) {
     spdlog::error("[error] on_global_remove received null data");
     return;
   }
-  auto* self = static_cast<PipewireGraph*>(data);
+  auto* self = static_cast<pipewire_graph*>(data);
   self->nodes_.erase(id);
   self->node_ports_.erase(id);
   self->links_.erase(id);
@@ -71,7 +84,7 @@ void PipewireGraph::on_global_remove(void* data, const uint32_t id) {
       self->all_nodes_.end());
 }
 
-bool PipewireGraph::initialize() {
+bool pipewire_graph::initialize() {
   std::lock_guard<std::mutex> lock(mutex_);
 
   if (initialized_) {
@@ -82,7 +95,7 @@ bool PipewireGraph::initialize() {
   // 1) Initialize PipeWire library (safe to call once)
   pw_init(nullptr, nullptr);
 
-  // 2) Create main loop, context, and core
+  // 2) Create the main loop, context, and core
   pw_thread_loop_ = pw_thread_loop_new("camera-loop", nullptr);
   if (!pw_thread_loop_) {
     spdlog::error("[CameraManager] failed to create pw_main_loop.");
@@ -142,7 +155,7 @@ bool PipewireGraph::initialize() {
   return true;
 }
 
-void PipewireGraph::shutdown() {
+void pipewire_graph::shutdown() {
   std::lock_guard<std::mutex> lock(mutex_);
 
   if (!initialized_) {
@@ -177,9 +190,9 @@ void PipewireGraph::shutdown() {
   spdlog::info("[PipewireGraph] Shutdown completed");
 }
 
-void PipewireGraph::handleNodeInfo(const uint32_t id,
-                                   const uint32_t version,
-                                   const spa_dict* props) {
+void pipewire_graph::handleNodeInfo(const uint32_t id,
+                                    const uint32_t version,
+                                    const spa_dict* props) {
   std::lock_guard lock(data_mutex_);
 
   NodeInfo node{};
@@ -201,12 +214,12 @@ void PipewireGraph::handleNodeInfo(const uint32_t id,
     audio_nodes_.push_back(node);
   }
 
-  printNodeInfo(node);
+  printDebugNodeInfo(node);
 }
 
-void PipewireGraph::handlePortInfo(const uint32_t id,
-                                   uint32_t /* version */,
-                                   const spa_dict* props) {
+void pipewire_graph::handlePortInfo(const uint32_t id,
+                                    uint32_t /* version */,
+                                    const spa_dict* props) {
   std::lock_guard lock(data_mutex_);
 
   PortInfo port{};
@@ -220,12 +233,12 @@ void PipewireGraph::handlePortInfo(const uint32_t id,
     node_ports_[port.node_id].push_back(port);
   }
 
-  printPortInfo(port);
+  printDebugPortInfo(port);
 }
 
-void PipewireGraph::handleLinkInfo(const uint32_t id,
-                                   uint32_t /* version */,
-                                   const spa_dict* props) {
+void pipewire_graph::handleLinkInfo(const uint32_t id,
+                                    uint32_t /* version */,
+                                    const spa_dict* props) {
   std::lock_guard lock(data_mutex_);
 
   LinkInfo link{};
@@ -248,49 +261,49 @@ void PipewireGraph::handleLinkInfo(const uint32_t id,
   links_[id] = link;
   active_links_.push_back(link);
 
-  printLinkInfo(link);
+  printDebugLinkInfo(link);
 }
 
-bool PipewireGraph::isCamera(const spa_dict* props) {
+bool pipewire_graph::isCamera(const spa_dict* props) {
   const char* media_class = spa_dict_lookup(props, PW_KEY_MEDIA_CLASS);
   return media_class &&
          (strstr(media_class, "Video/Source") || strstr(media_class, "Camera"));
 }
 
-bool PipewireGraph::isAudio(const spa_dict* props) {
+bool pipewire_graph::isAudio(const spa_dict* props) {
   const char* media_class = spa_dict_lookup(props, PW_KEY_MEDIA_CLASS);
   return media_class && (strstr(media_class, "Audio/Source") ||
                          strstr(media_class, "Audio/Sink"));
 }
 
-std::string PipewireGraph::getStringProperty(const spa_dict* props,
-                                             const char* key) {
+std::string pipewire_graph::getStringProperty(const spa_dict* props,
+                                              const char* key) {
   const char* value = spa_dict_lookup(props, key);
   return value ? std::string(value) : std::string();
 }
 
-const std::vector<NodeInfo>& PipewireGraph::getCameraNodes() const {
+const std::vector<NodeInfo>& pipewire_graph::getCameraNodes() const {
   std::lock_guard lock(data_mutex_);
   return camera_nodes_;
 }
 
-const std::vector<NodeInfo>& PipewireGraph::getAudioNodes() const {
+const std::vector<NodeInfo>& pipewire_graph::getAudioNodes() const {
   std::lock_guard lock(data_mutex_);
   return audio_nodes_;
 }
 
-const std::vector<NodeInfo>& PipewireGraph::getAllNodes() const {
+const std::vector<NodeInfo>& pipewire_graph::getAllNodes() const {
   std::lock_guard lock(data_mutex_);
   return all_nodes_;
 }
 
-const NodeInfo* PipewireGraph::getNodeById(const uint32_t id) const {
+const NodeInfo* pipewire_graph::getNodeById(const uint32_t id) const {
   std::lock_guard lock(data_mutex_);
   const auto it = nodes_.find(id);
   return (it != nodes_.end()) ? &it->second : nullptr;
 }
 
-const std::vector<PortInfo>& PipewireGraph::getPortsForNode(
+const std::vector<PortInfo>& pipewire_graph::getPortsForNode(
     const uint32_t node_id) const {
   std::lock_guard lock(data_mutex_);
   const auto it = node_ports_.find(node_id);
@@ -298,28 +311,28 @@ const std::vector<PortInfo>& PipewireGraph::getPortsForNode(
   return (it != node_ports_.end()) ? it->second : empty_vector;
 }
 
-const std::vector<LinkInfo>& PipewireGraph::getActiveLinks() const {
+const std::vector<LinkInfo>& pipewire_graph::getActiveLinks() const {
   std::lock_guard lock(data_mutex_);
   return active_links_;
 }
 
-void PipewireGraph::printNodeInfo(const NodeInfo& node) {
-  spdlog::info(
+void pipewire_graph::printDebugNodeInfo(const NodeInfo& node) {
+  spdlog::debug(
       "[PipewireGraph] Node ID: {} | Name: {} | Media Class: {} | Factory: {} "
       "| Version: {} | Camera: {} | Audio: {}",
       node.id, node.name, node.media_class, node.factory_name, node.version,
       node.is_camera, node.is_audio);
 }
 
-void PipewireGraph::printPortInfo(const PortInfo& port) {
-  spdlog::info(
+void pipewire_graph::printDebugPortInfo(const PortInfo& port) {
+  spdlog::debug(
       "[PipewireGraph] Port ID: {} | Node ID: {} | Name: {} | Direction: {} | "
       "Format: {}",
       port.id, port.node_id, port.name, port.direction, port.format);
 }
 
-void PipewireGraph::printLinkInfo(const LinkInfo& link) {
-  spdlog::info(
+void pipewire_graph::printDebugLinkInfo(const LinkInfo& link) {
+  spdlog::debug(
       "[PipewireGraph] Link ID: {} | Output Node: {} | Input Node: {} | Output "
       "Port: {} | Input Port: {}",
       link.id, link.output_node_id, link.input_node_id, link.output_port_id,

--- a/plugins/camera_pipewire/pipewire_graph.h
+++ b/plugins/camera_pipewire/pipewire_graph.h
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2020-2025 Toyota Connected North America
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 #ifndef PLUGINS_CAMERA_PIPEWIRE_PIPEWIREGRAPH_H
 #define PLUGINS_CAMERA_PIPEWIRE_PIPEWIREGRAPH_H
 
@@ -34,11 +50,11 @@ struct LinkInfo {
   uint32_t input_port_id;
 };
 
-class PipewireGraph {
+class pipewire_graph {
  public:
-  ~PipewireGraph();
-  PipewireGraph();
-  static PipewireGraph& instance();
+  ~pipewire_graph();
+  pipewire_graph();
+  static pipewire_graph& instance();
   pw_thread_loop* threadLoop() const { return pw_thread_loop_; }
   pw_core* core() const { return pw_core_; }
 
@@ -108,9 +124,9 @@ class PipewireGraph {
   bool initialized_ = false;
 
   // Debug/logging methods
-  static void printNodeInfo(const NodeInfo& node);
-  static void printPortInfo(const PortInfo& port);
-  static void printLinkInfo(const LinkInfo& link);
+  static void printDebugNodeInfo(const NodeInfo& node);
+  static void printDebugPortInfo(const PortInfo& port);
+  static void printDebugLinkInfo(const LinkInfo& link);
 };
 
 #endif  // PLUGINS_CAMERA_PIPEWIRE_PIPEWIREGRAPH_H

